src/csm_testing/tests/sat_functional/status/test_status.py

```diff
@@ -43,8 +43,17 @@ def get_header(command, num_lines_in_header):
     proc = subprocess.run(shlex.split(command), stdout=subprocess.PIPE, stderr=subprocess.PIPE,
                           check=True)
     status_output = proc.stdout.decode().strip()
+    status_err = proc.stderr.decode().strip()
     status_output_header = '\n'.join(status_output.splitlines()[:num_lines_in_header])
-    return status_output_header
+    return status_output, status_err, status_output_header
+
+def adjust_expected_header(header, sat_output):
+        new_header = header.copy()
+        for col in header:
+            info_line = f"INFO: All values for '{col}' are 'MISSING', omitting key."
+            if info_line in sat_output:
+                new_header.remove(col)
+        return new_header
```

Comment:
This is indented further than it should be.

```suggestion
    new_header = header.copy()
    for col in header:
        info_line = f"INFO: All values for '{col}' are 'MISSING', omitting key."
        if info_line in sat_output:
            new_header.remove(col)
    return new_header
```